src/components/OrganizationSelection/OrganizationSelectionContainer.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { debounce, get } from 'lodash';
+
+import OrganizationSelectionDisplay from './OrganizationSelectionDisplay';
+
+export default class OrganizationSelectionContainer extends React.Component {
+  static manifest = Object.freeze({
+    organizations: {
+      type: 'okapi',
+      path: 'erm/org',
+      limitParam: 'perPage',
+      perRequest: 100,
+      params: {
+        match: 'name',
+        term: '${orgNameFilter}', // eslint-disable-line no-template-curly-in-string
```

Comment:
Use `%{foo}` to substitute in a local resource's value. `${foo}` [has been deprecated](https://github.com/folio-org/stripes-connect/blob/master/doc/api.md#text-substitution) for a long time, and eslint doesn't get offended either.

Reply:
Oh my god I just realised that I've read that section of the API as the direct opposite of what it actually says so many times...